components/action-mgt/org.wso2.carbon.identity.action.mgt/src/main/java/org/wso2/carbon/identity/action/mgt/ActionManager.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.action.mgt;
+
+import org.wso2.carbon.identity.action.mgt.exception.ActionMgtException;
+import org.wso2.carbon.identity.action.mgt.model.Action;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Action Manager Interface.
+ */
+public interface ActionManager {
```

Comment:
Let's make this ActionManagementService

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/5771/commits/94e90e62016b434a0f37aa2b1f7ff0edc34bb64f